streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -335,41 +332,37 @@ private void prepareStreamThread(final StreamThread thread,
                 "",
                 Collections.emptySet(),
                 Collections.emptySet()
-            )
-        ).anyTimes();
-        EasyMock.expect(thread.waitOnThreadState(EasyMock.isA(StreamThread.State.class), anyLong())).andStubReturn(true);
-        EasyMock.expect(thread.isAlive()).andReturn(true).times(0, 1);
-        thread.resizeCache(EasyMock.anyLong());
-        EasyMock.expectLastCall().anyTimes();
-        thread.requestLeaveGroupDuringShutdown();
-        EasyMock.expectLastCall().anyTimes();
-        EasyMock.expect(thread.getName()).andStubReturn("processId-StreamThread-" + threadId);
-        thread.shutdown();
-        EasyMock.expectLastCall().andAnswer(() -> {
+        ));
+        when(thread.waitOnThreadState(isA(StreamThread.State.class), anyLong())).thenReturn(true);
+        when(thread.isStateAlive()).thenReturn(true);
+        verify(thread, atMostOnce()).isStateAlive();
```

Comment:
It seems to me this check is no-op since the thread is not ready to be used. Hence, the execution times is always 0.